stockflux-watchlist/src/components/Watchlist/Watchlist.js

```diff
@@ -68,10 +69,35 @@ const Watchlist = () => {
     if (context && currentListener === latestListener) {
       const newSymbol = context.id.default;
       setWatchlist(getDistinctElementArray([newSymbol, ...watchlist]));
+      displayNotification(newSymbol);
     }
   });
   latestListener = currentListener;
 
+  const displayNotification = newSymbol => {
+    const alreadyInWatchlist = watchlist.includes(newSymbol);
+    const watchlistName = name.slice().toUpperCase();
+    if (alreadyInWatchlist) {
```

Comment:
This might be more succinct than the if-else:

```
showNotification({
        message: {
          symbol: newSymbol,
          watchlistName: name.toUpperCase(),
          alreadyInWatchlist,
          messageText: ` ${alreadyInWatchlist ? 'moved' : 'added'} to the top`
        }
      });
```

Reply:
Absolutely, thanks for noticing, fixed -> 27269452380451083d247e8139fb6ca741e28185